src/movies.js

```diff
@@ -1,22 +1,108 @@
+const movies = require("./data");
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors() {}
+function getAllDirectors(obj) {
+  let directores=[];
+  obj.map((obj)=>{
+      directores.push(obj.director)
+  })
+  return directores;
+}
+getAllDirectors(movies);
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
-function howManyMovies() {}
+function howManyMovies(arr) {
+let contadorSteven=0;
+let numpelis= arr.filter((peli)=>
+  peli.director==='Steven Spielberg' && peli.genre.includes('Drama'));
+contadorSteven=numpelis.length
+return contadorSteven
```

Comment:
Con esto quitamos dos lineas de codigo 😉 
```suggestion
return numpelis.length
```